frontend/front/src/pages/Public/Pages/AboutUs/AboutUs.js

```diff
@@ -60,77 +52,43 @@ function AboutUs() {
               </Link>
               <Box>
                 <Typography variant="body">
-                  <Link
-                    href="https://www.ulem.org/"
-                    sx={{
-                      color: "inherit",
-                      textDecoration: "none",
-                    }}
-                    target="_blank"
-                    rel="noopener"
-                  >
-                    <b>The Urban League of Eastern Massachusetts (ULEM)</b>
-                  </Link>{" "}
-                  is the sponsor of the BHPA effort. Since 1919, ULEM has
-                  delivered workforce and economic development services and
-                  programs to increase self-reliance of residents of the Boston
-                  community and surrounding metropolitan areas. ULEM is a 501c3
-                  nonprofit organization and one of the oldest affiliates within
-                  the National Urban League movement.{" "}
+                  <Trans i18nKey="public.about-us.ulem1">
+                    The
+                    <Link
```

Comment:
How does the library handle react inside the <Trans> tag? Seems like this would break languages that have different word order or something.